### ISSUE TITLE
Perform quality control data related to modality-fatality numbers on VZV

### ISSUE THREAD

**Issue by frankhereford** [cityofaustin/atd-data-tech#7842]
### Describe the bug

Xavier A. has reported: 

> Sorry for the super late notification on this one, but I am looking the [VZ Viewer Preview](https://atd-vzv-preview.netlify.app/) and noticed that the modes are off by 1 Pedestrian Killed.

I have asked him the following: 

> Which is the reference value that it should match? I can’t recall if it’s a query, the VZE or maybe the FRB spreadsheet.

I'll update the issue with any new information. @patrickm02L, I'm only adding you so it's there for when you get back - no worries. @amenity, please let me know if you have any direction on what I can do to help. Thanks y'all.

### Solution
- Make data match Xavier's
- Track all the crash IDs we are changing to ☝🏻 
- Discuss in next Vision Zero Sprint Review

#### How would I rate the impact of this issue?
- [ ] Severe — cannot perform work, no workaround
- [ ] Major — can only perform work using a workaround
- [ ] Minor  — can perform work, but could be easier or faster
- [ ] Does not affect work performance
- [x] Unsure

**Comment by frankhereford** [comment]
## TLDR
According to the documents which I have been provided by Xavier A, for fatalities YTD through Oct-31, the VZV is ± 4 on the following modalities: Pedestrian, Bicyclists, & Motorists (excluding Motorcycles). 


## Details
The Vision Zero team maintains a pair of spreadsheets which they use to compute statistics to compare to the VZV's computed statistics. While close, the VZV's numbers do not match the VZ team's calculations. 

I spoke with Xavier A. this afternoon via teams to find out more about what's going on here. 

The source of truth to which the VZV is being compared to is the Fatality Review Board spreadsheet which is maintained by the VZ team, primarily by Joel M. and Xavier A. I currently have access to this document via OneDrive, and while it can not be shared here due to PII concerns, please contact me via slack if you'd like to see the raw data VZ's numbers are coming from. 

This FRB spreadsheet is the source of verified, curated data related to fatalities. This data is then fed into another spreadsheet which is maintained by Xavier A, which computes a wide variety of different statistics which are in turn used to compare against the VZV. Xavier has provided me a copy of this second spreadsheet as well, which I can make available to this issues assignee and others as needed - please slack me.

## Opinion

I think, very broadly, we are dealing with imperfect data from CRIS which we use to compute statistics for. The Vision Zero team uses this same data, but puts a lot of effort into cleaning up details about high-severity crashes (both in the VZDB and in their internal documents). Both the VZV application and the Vision Zero team have similar, but not identical queries or criteria used to compute statistics, and accordingly they do not always match. I believe the larger issue would benefit from a discussion to see if it's possible to define exactly how each statistic is calculated to increase our chances of matching our statistics' values.

## Possible short term solution
We can ID which rows are being picked up (or not) in our queries and massage the data to make the queries come up with the right numbers. This feels like a band-aid, but it may fit the bill given the situation -- I look forward to team conversation on how to move forward. 

Unfortunately, it seems there are no more permanent solutions than temporary ones, and I am not sure if this is in the interest of long term maintainability.

**Comment by amenity** [comment]
Thanks so much for intaking this one, @frankhereford ... @patrickm02L - pulling this on deck assuming that would be your wish. ;)

**Comment by patrickm02L** [comment]
To @frankhereford's point I agree that this is a larger issue. We've identified several instances where the numbers are off, and have brought this up in multiple Vision Zero monthly meetings. 

Rather than kick the bucket down the road by continuing to do one-off updates, I would like to make this a priority before trying to build out other features. I'll collect the issues that have data control problems and inconsistencies into an Epic, get clarity around what we'll need to straighten out the queries and get consistent calculations, and then bring this up at the Dec 7th Vision Zero meeting. 

Let's update the VZV pedestrians killed number manually until then.

**Comment by mateoclarke** [comment]
We created this report page in VZE for Admin users to help us identify inconsistencies in KSI counts. Have we referred to this for this specific data quality issue?
https://visionzero.austin.gov/editor/#/reports/inconsistent_ksi_counts

**Comment by frankhereford** [comment]
> Let's update the VZV pedestrians killed number manually until then.

@patrickm02L - you got it. I plan on doing it in this way:

1) Communicating with Xavier to confirm that the numbers from the documents he gave me are 100% correct.
1) Figure out what crashes need to be adjusted and in what way.
1) Record all the crashes which will be changed and how.
1) Execute the updates and confirm that the VZV is showing good data.

In the course of doing this work, I will be kicking off some manual invocations of our ETL tool to update the `development` versions of the two Socrata datasets. If anyone has concerns about working on those datasets during the work-day, please let me know.

**Comment by frankhereford** [comment]
I looked into this today, with the hope that I would be able to identify a mismatch in the FRB spreadsheet data and the [information stored in the Socrata dataset](https://data.austintexas.gov/Transportation-and-Mobility/Vision-Zero-Crash-Report-Data/y2wy-tgr5). I have correlated the crashes (by case id) between the two for crashes involving pedestrians, and I have not been able to identify the particular crash record which stood out as in one data set and not the other.

As a result, I do not believe that I understand the problem correctly. I do observe that the total number of fatalities shown in the VZV in the total fatality card is off by one compared to the sum of fatalities in the "By Travel Mode" table, but if that is the off-by-one error in question, I do not understand how it is known that it is a pedestrian fatality which is at the root of the mismatch.

Also, I have observed that the "VZLC Analysis" spreadsheet's mode numbers do not align with the VZV for the modes of Pedestrian or Bicyclist (for October), so I am not sure if that is the value mismatch either.

**Comment by patrickm02L** [comment]
@frankhereford thanks for the update.

I'm going to bring this back to the VZ team after the end-of-the-year break because I think we need more clarity, and I want to offer up a larger conversation that addresses a lot of related data quality issues #7866.

**Comment by patrickm02L** [comment]
Notes from meeting with Mateo 1/7/22:

What format is the Fatality Review Board data? Can it get moved into vision zero editor? 

By making it part of VZE so that we can automatically associate FRB crashes to existing crashes, or present together.

This is getting into the realm of creating a new feature. We need to be able to talk about highlighting all the fatalities one by one, and we need extra data about each of them.

**Comment by frankhereford** [comment]
> What format is the Fatality Review Board data? Can it get moved into vision zero editor?

It is currently stored in a spreadsheet which is maintained by the Vision Zero team. You may want to reach out to Joel to get access to it.

**Comment by patrickm02L** [comment]
Data Quality for modality-fatality has been resolved through Issue #8424.
